src/index.ts

```diff
@@ -20,11 +20,6 @@ import move from './actions/move.js';
 import commit from './actions/commit.js';
 import dump from './actions/dump.js';
 
-export type { AppendOptions } from './actions/append.js';
-export type { CopyOptions } from './actions/copy.js';
-export type { CopyAsyncOptions } from './actions/copy-async.js';
-export type { MemFsEditorFileDump } from './actions/dump.js';
```

Comment:
Breaking change. I prefer library user to infer type structure rather than exposing types interfaces we need to maintain over time.